### ISSUE TITLE
20s delay when clicking `Reauthenticate` on Android

### ISSUE THREAD

**Issue by szmarczak** [tailscale/tailscale#13375]
### What is the issue?

https://github.com/user-attachments/assets/5898ca72-2f9a-42d5-9235-c36b6b006d8d

If you see `Video can't be played because the file is corrupt` above, use VLC:

1. Media -> Open Network Stream
2. https://github.com/user-attachments/assets/5898ca72-2f9a-42d5-9235-c36b6b006d8d

### Steps to reproduce

See above.

### Are there any recent changes that introduced the issue?

None that I know of.

### OS

Android

### OS version

Android 14

### Tailscale version

1.72

### Other software

N/A

**Comment by bradfitz** [comment]
Yeah, that's a valid bug. The UI should indicate that an operation is in flight.

As for 20s, it's likely that subsequent presses are restarting the operation, delaying the result. If the button were disabled after the first click, it'd likely be faster.

**Comment by szmarczak** [comment]
No, I can reproduce without spamming the button. Just a single tap.

**Comment by bradfitz** [comment]
If you could repro again (ideally with a single click) and then capture a bug report from the app and share its identifier here, that'd help us see what was happening.

**Comment by bradfitz** [comment]
> However, most of the time it works ok. Just sometimes it fails like this.

It's possible you got unlucky and we were redeploying a backend server at the time. But we should fix the UI in any case.

**Comment by szmarczak** [comment]
> > However, most of the time it works ok. Just sometimes it fails like this.
> 
> It's possible you got unlucky and we were redeploying a backend server at the time. But we should fix the UI in any case.
> 

Possibly. Or maybe just my ISP. I can no longer reproduce this, now it just opens within 3s. I assume the in-app browser view is shown when the page is already loaded? If so I think it would be better to show it immediately so the user is aware that it is loading.

**Comment by dblohm7** [comment]
It's highly likely that the delay was due to a control plane restart. Leaving this open to address the UX issue.
